internal/v4/relations_test.go

```diff
@@ -226,6 +226,7 @@ var metaCharmRelatedTests = []struct {
 				Meta: map[string]interface{}{
 					"id-name": params.IdNameResponse{"wordpress"},
 					"charm-metadata": &charm.Meta{
+						Format: 1,
```

Comment:
It's a strange thing related to fixing yaml marshaling for charm.Meta. With the marshaling not fixed, the format field was getting written out explicitly (value 0), which meant that when we read the charm it did not acquire its default value (1 for some historical reason) but zero. Now that we're not writing the Format field (which is correct I think - Format is an ancient relic) we start seeing the usual default value of Format.